cli.js

```diff
@@ -1,13 +1,26 @@
 #!/usr/bin/env node
 
 var chalk = require("chalk");
-var lib = require(".");
+var argv = require("yargs").argv;
+var Changelog = require(".").Changelog;
+var ConfigurationError = require(".").ConfigurationError;
 
-var Changelog = lib.Changelog;
-var ConfigurationError = lib.ConfigurationError;
+if (argv.help) {
```

Comment:
If you _are_ going to use `yargs`, seems like you could do some minimal [configuration](https://github.com/yargs/yargs#optionskey-opt) and get this [`--help`](https://github.com/yargs/yargs#help) output for free (as well as `help` subcommand).

```js
var argv = require("yargs")
  .usage("$0 [options]")
  .options({
    "tag-from": {
      type: "string",
      desc: "A git tag that determines the lower bound of the range of commits (defaults to last available)"
    },
    "tag-to": {
      type: "string",
      desc: "A git tag that determines the upper bound of the range of commits"
    }
  })
  .help()
  .argv;
```

Given the powers of yargs, you might even consider the tagFrom and tagTo to be [positional arguments](https://github.com/yargs/yargs#positional-arguments) rather than flags, given that it really only makes sense to specify them in the order one might expect when specifying the range in git (`${fromTag}..${toTag}`).